libs/src/services/creatorNode/CreatorNodeSelection.js

```diff
@@ -211,6 +215,10 @@ class CreatorNodeSelection extends ServiceSelection {
 
     return { healthyServicesList, healthyServicesMap: servicesMap }
   }
+
+  _hasEnoughStorageSpace ({ storagePathSize, storagePathUsed }) {
+    return Math.round(100 * (storagePathUsed / storagePathSize)) < this.maxStoragePathUsage
```

Comment:
Maybe worth doing Math.ceil

Reply:
On second thought why when round?

Reply:
I was thinking about how it'd be easier to read the availability... but we don't use it for logging purposes lol. i'll update to use the raw number